hack/build.sh

```diff
@@ -23,6 +23,8 @@ set -o pipefail
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${REPO_ROOT}/hack/util.sh"
 
+LDFLAGS="$(util::version_ldflags) ${LDFLAGS:-}"
```

Comment:
this is confusing

Reply:
LDFLAGS="${LDFLAGS}:-$(util::version_ldflags)}" ? should be like this?

Reply:
It's not a mistake. Users can set custom `ldflags` by `LDFLAGS` variable., not only version info. FYI:
https://github.com/kubernetes/kubernetes/blob/a2408f39c7233fc7ceb06795ffd24b84ca471243/hack/lib/golang.sh#L847

Reply:
/lgtm